packages/core/upload/admin/src/components/AssetDialog/BrowseStep/index.jsx

```diff
@@ -374,7 +374,7 @@ export const BrowseStep = ({
         </>
       )}
 
-      {pagination.pageCount > 0 && (
+      {pagination.total > pagination.pageSize && (
```

Comment:
what's happening here (and with the other pagination changes)?

Reply:
uh? I just updated the branch with develop and it brought this, no idea where this comes from :thinking:

Reply:
https://github.com/strapi/strapi/pull/21495/commits

Reply:
weird, maybe you need to rebase